examples/HikariExample/src/main/java/software/amazon/HikariExample.java

```diff
@@ -59,9 +59,9 @@ public static void main(String[] args) throws SQLException {
     // Try and make a connection:
     try (final Connection conn = ds.getConnection();
         final Statement statement = conn.createStatement();
-        final ResultSet rs = statement.executeQuery("SELECT * FROM employees")) {
+        final ResultSet rs = statement.executeQuery("SELECT * from aurora_db_instance_identifier()")) {
```

Comment:
nit: Can we replace * with some field name?

Reply:
This function only has 1 field, I am okay with *